### PR TITLE
Magiclysm: Spells make sounds when cast

### PR DESCRIPTION
**Pull request by KorGgenT**
#### Summary

```SUMMARY: Mods "Spells make sounds when cast"```
#### Purpose of change

This is a pretty obviously important balancing factor for spells.
I also added a few dormant flags to be worked on in an upcoming PR.
#### Describe the solution

Spells with the "VERBAL" flag cause the player to make noise equal to their shout volume / 2 at their position.
Spells without the SILENT flag make noise at their target(s) equal to damage / 3
Spells with the LOUD flag make noise at their target(s) equal to 2 * damage / 3

### CODE REVIEW

**Comment by dkargin** [comment]
That sounds great!

**Comment by KorGgenT** [comment]
> That sounds great!

i see what you did there

File: data/mods/Magiclysm/Spells/biomancer.json [hunk 1]
```diff
@@ -157,8 +164,9 @@
     "duration_increment": 1,
     "difficulty": 0,
     "max_level": 1,
+    "flags": [ "NO_LEGS", "CONCENTRATE" ],
```
Comment (ZhilkinSerg) [inline data/mods/Magiclysm/Spells/biomancer.json]:
```suggestion
```

File: data/mods/Magiclysm/Spells/druid.json [hunk 2]
```diff
@@ -163,6 +169,7 @@
     "name": "Cause Bear",
     "description": "This spell appears to be very smudged.  You're fairly sure the name should be Cause Fear, but you're also fairly sure it won't have the desired effect because the instructions are hardly legible.  No time like the Cataclysm to find out, though!",
     "valid_targets": [ "ground" ],
+    "flags": [ "CONCENTRATE", "SOMATIC", "VERBAL", "NO_LEGS" ],
```
Comment (ZhilkinSerg) [inline data/mods/Magiclysm/Spells/druid.json]: There is another `flags` node in the same object below.

**Pull request merged**